File: src/easy/UglyNumber.swift
func uglyNumber(_ n: Int)-> Bool {
	if n <= 0 { return false }
	var primes = [2,3,5]
	var num = n
	for prime in primes {
		while num % prime == 0 {
			num/=prime
		}
		
	}
	return num == 1
}

File: lib/easy/ugly_number.dart
bool uglyNumber(num n) {
  if (n < 1) return false;
  List<num> primes = [2, 3, 5];
  for (var prime in primes) {
    while (n % prime == 0) {
      n ~/= prime;
    }
  }
  return n == 1;
}
